The example VM fails to provision on the 1.1.0 release - ImportError: No module named SCons.Script

```
$ git checkout 1.1.0
$ cd examples
$ vagrant up
```


>TASK [azavea.statsite : Build Statsite] ****************************************
fatal: [default]: FAILED! => {"changed": true, "cmd": ["make"], "delta": "0:00:00.139560", "end": "2017-03-29 20:21:14.317573", "failed": true, "rc": 2, "start": "2017-03-29 20:21:14.178013", "stderr": "Traceback (most recent call last):\n  File \"/usr/local/bin/scons\", line 190, in <module>\n    import SCons.Script\nImportError: No module named SCons.Script\nmake: *** [build] Error 1", "stdout": "scons statsite\nSCons import failed. Unable to find engine files in:\n  /usr/local/bin/../engine\n  /usr/local/bin/scons-local-2.5.1\n  /usr/local/bin/scons-local\n  /usr/local/lib/python2.7/dist-packages/lib/scons-2.5.1\n  /usr/local/lib/scons-2.5.1\n  /usr/lib/scons-2.5.1\n  /usr/local/lib/scons-2.5.1\n  /usr/local/lib/python2.7/dist-packages/lib/python2.7/site-packages/scons-2.5.1\n  /usr/local/lib/python2.7/site-packages/scons-2.5.1\n  /usr/lib/python2.7/site-packages/scons-2.5.1\n  /usr/local/lib/python2.7/site-packages/scons-2.5.1\n  /usr/lib/scons-2.5.1\n  /usr/local/lib/python2.7/dist-packages/lib/scons\n  /usr/local/lib/scons\n  /usr/lib/scons\n  /usr/local/lib/scons\n  /usr/local/lib/python2.7/dist-packages/lib/python2.7/site-packages/scons\n  /usr/local/lib/python2.7/site-packages/scons\n  /usr/lib/python2.7/site-packages/scons\n  /usr/local/lib/python2.7/site-packages/scons\n  /usr/lib/scons", "stdout_lines": ["scons statsite", "SCons import failed. Unable to find engine files in:", "  /usr/local/bin/../engine", "  /usr/local/bin/scons-local-2.5.1", "  /usr/local/bin/scons-local", "  /usr/local/lib/python2.7/dist-packages/lib/scons-2.5.1", "  /usr/local/lib/scons-2.5.1", "  /usr/lib/scons-2.5.1", "  /usr/local/lib/scons-2.5.1", "  /usr/local/lib/python2.7/dist-packages/lib/python2.7/site-packages/scons-2.5.1", "  /usr/local/lib/python2.7/site-packages/scons-2.5.1", "  /usr/lib/python2.7/site-packages/scons-2.5.1", "  /usr/local/lib/python2.7/site-packages/scons-2.5.1", "  /usr/lib/scons-2.5.1", "  /usr/local/lib/python2.7/dist-packages/lib/scons", "  /usr/local/lib/scons", "  /usr/lib/scons", "  /usr/local/lib/scons", "  /usr/local/lib/python2.7/dist-packages/lib/python2.7/site-packages/scons", "  /usr/local/lib/python2.7/site-packages/scons", "  /usr/lib/python2.7/site-packages/scons", "  /usr/local/lib/python2.7/site-packages/scons", "  /usr/lib/scons"], "warnings": []}

--- a/examples/roles.yml
+++ b/examples/roles.yml
@@ -1,4 +1,4 @@
 - name: azavea.pip
-  version: 0.1.1
+  version: 1.0.0
 - name: azavea.git
   version: 0.1.0

--- a/examples/site.yml
+++ b/examples/site.yml
@@ -1,4 +1,9 @@
 ---
 - hosts: all
+
+  pre_tasks:
+    - name: "Update APT Cache"
+      apt: update_cache=true
+
   roles:
     - { role: "azavea.statsite" }

--- a/tasks/main.yml
+++ b/tasks/main.yml
@@ -24,7 +24,7 @@
 
 - name: Install SCons
   pip: name="SCons"
-       extra_args="--egg"
+       extra_args="--egg --install-option --standard-lib"
        state=present
 
 - name: Build Statsite

